src/tests/ftest/datamover/posix_meta_entry.yaml

```diff
@@ -21,6 +21,8 @@ container:
   type: POSIX
   control_method: daos
 dcp:
+  env_vars:
+    - D_LOG_MASK=INFO,DFS=DEBUG
```

Comment:
We want to keep this turned on?

Reply:
Yes, DFS is quiet at debug anyway so the logs produced by this test are appropriate.